pylibgt/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "cmake>=2.18.1", "scikit-build", "ninja"]
```

Comment:
Does this package actually need to build anything with scikit-build? I thought it's only providing headers, which can be done with normal setuptools through the MANIFEST.in.

Reply:
It's header only but gt4py-declarative requires the installed CMake config files. It's either scikit-build or running CMake manually.

Reply:
Actually it turned out to be better to make calling `CMake` a pre-step, as this allows us to build a pure-python wheel (platform independent).